complete.go

```diff
@@ -124,6 +124,8 @@ var (
 		"dirfirst!",
 		"dironly",
 		"dirpreviews",
+		"nodirpreviews",
+		"dirpreviews!",
 		"nodironly",
 		"dironly!",
```

Comment:
nit: move nodironly and dironly! up three lines, after dironly

Reply:
That's fair, they are sorted properly now